hazelcast/src/main/java/com/hazelcast/internal/monitor/impl/LocalMapStatsImpl.java

```diff
@@ -84,28 +114,27 @@
             newUpdater(LocalMapStatsImpl.class, "maxRemoveLatency");
 
     private final ConcurrentMap<String, LocalIndexStatsImpl> mutableIndexStats =
-            new ConcurrentHashMap<String, LocalIndexStatsImpl>();
-    private final Map<String, LocalIndexStats> indexStats = Collections.<String, LocalIndexStats>unmodifiableMap(
-            mutableIndexStats);
+            new ConcurrentHashMap<>();
+    private final Map<String, LocalIndexStats> indexStats = Collections.unmodifiableMap(mutableIndexStats);
 
     // These fields are only accessed through the updaters
-    @Probe
+    @Probe(name = MAP_METRIC_LAST_ACCESS_TIME)
```

Comment:
How about `MS` unit here and in other time related probes in this class?

Reply:
Done